src/HPWHpresets.cc

```diff
@@ -1661,10 +1661,18 @@ int HPWH::HPWHinit_presets(MODELS presetNum) {
 			3.993147, 3.713376, 3.616836, 3.710957, 3.470484, 3.264466, 3.14959
 			});
 
-		// Set up regular grid interpolator. 
+		// Set up regular grid interpolator.
+#if 0
 		Btwxt::GriddedData gridded_data(compressor.perfGrid, compressor.perfGridValues);
 		gridded_data.set_axis_extrap_method(2, Btwxt::Method::LINEAR); //Linearly extrapolate on Tin (F)
-		compressor.perfRGI = new Btwxt::RegularGridInterpolator(gridded_data);
+#endif
+		Btwxt::GridAxis g0(compressor.perfGrid[0], "TAir", Btwxt::InterpolationMethod::linear, Btwxt::ExtrapolationMethod::linear);
+		Btwxt::GridAxis g1(compressor.perfGrid[1], "TOut", Btwxt::InterpolationMethod::linear, Btwxt::ExtrapolationMethod::linear);
+		Btwxt::GridAxis g2(compressor.perfGrid[2], "TIn", Btwxt::InterpolationMethod::linear, Btwxt::ExtrapolationMethod::linear);
```

Comment:
Previous version specified linear extrapolation on axis two, but let the others default. @tanaya-mankad is going to verify what those defaults were.

Reply:
The GridAxis default used to be CONSTANT extrapolation, so there's definitely a change here.

Reply:
Changed TAir and TOut to constant.